user.js

```diff
@@ -318,6 +318,7 @@ user_pref("browser.startup.homepage_override.buildID",		"20100101");
 // PREF: Prevent font fingerprinting
 // https://browserleaks.com/fonts
 // https://github.com/pyllyukko/user.js/issues/120
+// NOTE: Will result in almost all webpages being ugly. If you don't want this, but don't want font fingerprinting, use something like FluxFonts.
```

Comment:
It's `NOTICE`, not `NOTE`. Also to have it on `README.md`, you need to run `make whatdoesitdo`. Also also... they are not ugly once you get used to it :grin: 

Or was your intention to have it only on `user.js`?